app/helpers/application_helper.rb

```diff
@@ -201,4 +201,8 @@ def tag_url(tag, page)
   def user_url(user)
     URL.user(user)
   end
+
+  def sanitized_referer(referer)
```

Comment:
side question: is this helper used anywhere?

Reply:
good catch! It should be used in the view but the view is referencing the `URL.`  method directly

Reply:
Haha, I added it to make the view nicer (and to follow the pattern we established elsewhere), but then didn't actually use it. I'll change that now.